services/Cargo.toml

```diff
@@ -22,3 +22,4 @@ pwhash = "0.3"
 serde_urlencoded = "0.6"
 futures = "0.3"
 image = "0.23"
+reqwest = "0.10.8" # TODO: dev-dependency?
```

Comment:
Yes, since you only use it in an example for now. We can move it back if we do Http requests from within with reqwest.

Reply:
92699a3d13dbdd2eb312e95b07ae9ec5bc45c9dd